components/contact/ContactForm.jsx

```diff
@@ -181,14 +203,15 @@ function ContactForm() {
     validateForm]);
 
   return (
-    <form id="contact-form" onSubmit={handleSubmit}>
+    <form className={classes.form} id="contact-form" onSubmit={handleSubmit}>
       <Grid container alignItems="center" justifyContent="center" direction="column">
         <Grid container alignItems="center" justifyContent="center" direction="row" spacing={2}>
           <Grid item xs={6}>
+            <InputLabel htmlFor='contact-firstname' className={classes.formLabel}>First Name*</InputLabel>
```

Comment:
not sure if this was a design decision i missed but i think i liked how it looked with the space next to the asterisk more

Reply:
I believe it was a design choice.  After I checked the figma file they did not have any space next to the asterisk.